<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_listar_despesas</class>
 <widget class="QDialog" name="tela_listar_despesas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Despesas</string>
  </property>
  <widget class="QGroupBox" name="gb_buscar">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>390</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="title">
    <string>Buscar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="lb_nome">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>17</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Período de:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_buscar">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>15</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
    <property name="icon">
     <iconset resource="imagens.qrc">
      <normaloff>:/img/img/buscar.png</normaloff>:/img/img/buscar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QDateEdit" name="data_inicial">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>17</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>3000</year>
      <month>12</month>
      <day>31</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2000</year>
      <month>9</month>
      <day>14</day>
     </datetime>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="timeSpec">
     <enum>Qt::LocalTime</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lb_a">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>17</y>
      <width>7</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="data_final">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>17</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>3000</year>
      <month>12</month>
      <day>31</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2000</year>
      <month>9</month>
      <day>14</day>
     </datetime>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="timeSpec">
     <enum>Qt::LocalTime</enum>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tw_despesas">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>79</y>
     <width>740</width>
     <height>416</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>360</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
  </widget>
  <widget class="QGroupBox" name="gb_filtrar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>344</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="title">
    <string>Filtrar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="btn_filtrar">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>15</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Filtrar</string>
    </property>
    <property name="icon">
     <iconset resource="imagens.qrc">
      <normaloff>:/img/img/filtrar.png</normaloff>:/img/img/filtrar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lb_codigo">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>17</y>
      <width>42</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Código:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_codigo">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>17</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMask">
     <string>000000000; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="lb_status">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>17</y>
      <width>38</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_status">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>17</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Todos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aberta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Atrasada</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fechada</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_cancelar">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>90</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/1345902747_button_cancel.png</normaloff>:/img/img/1345902747_button_cancel.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_limpar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>90</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Limpar</string>
   </property>
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/img/img/limpar.png</normaloff>:/img/img/limpar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>data_inicial</tabstop>
  <tabstop>data_final</tabstop>
  <tabstop>btn_buscar</tabstop>
  <tabstop>btn_limpar</tabstop>
  <tabstop>btn_cancelar</tabstop>
  <tabstop>tw_despesas</tabstop>
 </tabstops>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
